soccer/planning/MotionInstant.hpp

```diff
@@ -16,10 +16,6 @@ struct MotionInstant {
     Geometry2d::Point pos;
     Geometry2d::Point vel;
 
-    friend bool operator==(const MotionInstant& a, const MotionInstant& b) {
```

Comment:
I didn't think we really needed it. Its not used anywhere and its not very useful for anything since you Point uses exact Point comparisons.